packages/http/src/mocker/callback/scheduler.ts

```diff
@@ -0,0 +1,45 @@
+import { IHttpCallbackOperation, IHttpOperationRequest } from '@stoplight/types';
+import { IHttpOperationConfig, IHttpRequest, IHttpResponse } from '../../types';
+import withLogger from '../../withLogger';
+import { runCallback } from './runner';
+import { RangeOrNumber } from '@stoplight/prism-core';
+import { pipe } from 'fp-ts/lib/pipeable';
+import * as Option from 'fp-ts/lib/Option';
+
+export function scheduleCallback({
+                                   callback,
+                                   request,
+                                   response,
+                                   config,
+                                 }: {
```

Comment:
Since all these parameters are all mandatory I think it might make sense to use regular function arguments
`scheduleCallback(callback, request,response,config)`